ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+  Thanks for your interest in using the App Center SDK for React Native.
+
+  If your issue is not about the Analytics/Crashes/Push SDK for React Native,
+  (for example a build issue and you are not using our SDK, or your issue is about CodePush),
```

Comment:
May be we can phrase it like this -

If you have an issue in React Native platform but **have not** integrated App Center SDK in your application, please contact support by clicking on the blue icon on the lower right of App Center portal (https://appcenter.ms). Our support team will get in touch to assist you.